sklearn/_build_utils/deprecated_modules.py

```diff
@@ -0,0 +1,37 @@
+"""Generates submodule to allow deprecation of submodules and keeping git
+blame."""
+from pathlib import Path
+
+
+_DEPRECATED_MODULES = {
+    # TODO: Remove in 0.24
```

Comment:
The entire file should be removed in 0.24 right? (ideally)

Reply:
That is the ideal case. ;)